src/app/editor/editor.module.ts

```diff
@@ -0,0 +1,9 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { FormsModule, ReactiveFormsModule } from '@angular/forms';
+
+@NgModule({
+    declarations: [],
```

Comment:
Should we add EditorComponent to declarations?

Reply:
Sure, just didn't complete it that's why didn't add it